.github/workflows/main.yml

```diff
@@ -108,18 +111,29 @@ jobs:
           cd .. && mv quay-operator/bundle/manifests operators/project-quay/${{ github.event.inputs.tag }} && \
           rm -rf quay-operator operators/project-quay/${{ github.event.inputs.tag }}/quay-operator.service.yaml
 
+      - name: Get and Set Old Version
+        run: echo OLD_VERSION=$(yq -P '.channels[] | select(.name == "${{ github.event.inputs.channel }}") .currentCSV' operators/project-quay/project-quay.package.yaml) >> $GITHUB_ENV
+
       - name: Update CSV Image Versions
         uses: mikefarah/yq@master
         with:
           cmd: |
             yq eval -i '
-              del(.spec.replaces) |
+              .spec.replaces = "${{ env.OLD_VERSION }}" |
               .metadata.annotations.containerImage = "quay.io/projectquay/quay-operator:${{ github.event.inputs.tag }}" |
               .spec.install.spec.deployments[0].spec.template.spec.containers[0].image = "quay.io/projectquay/quay-operator:${{ github.event.inputs.tag }}" |
               .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[] |= select(.name == "RELATED_IMAGE_COMPONENT_BUILDER") .value = "quay.io/projectquay/quay-builder:${{ github.event.inputs.tag }}" |
               .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[] |= select(.name == "RELATED_IMAGE_COMPONENT_CLAIR") .value = "quay.io/projectquay/clair:${{ github.event.inputs.clair-version }}"
               ' operators/project-quay/${{ github.event.inputs.tag }}/quay-operator.clusterserviceversion.yaml
 
+      - name: Update Package currentCSV
+        uses: mikefarah/yq@master
+        with:
+          cmd: |
+            yq eval -i '
+              .channels[] |= select(.currentCSV == "${{ env.OLD_VERSION }}") .currentCSV' = "quay-operator.v${{ github.event.inputs.tag }}"
```

Comment:
`.currentCSV'` should just be `.currentCSV`

That single quote at the end breaks the yq command for me. Also, once the command is fixed by removing the single quote, the command overwrites the values for both the stable and preview channels. Since we are now taking this as an input, shouldn't we take this into account?